pkg/router/template/router.go

```diff
@@ -512,12 +512,27 @@ func (r *templateRouter) writeConfig() error {
 		r.state[k] = cfg
 	}
 
+	data, err := ioutil.ReadFile("/proc/sys/net/ipv4/neigh/default/gc_thresh3")
+	if err != nil {
+		glog.Warningf("Error reading ARP neighbour information: %s", err)
+	}
+	arpcache, err := strconv.Atoi(string(data[:len(data)-1]))
+	if err != nil {
+		glog.Warningf("Error: %s", err)
+	}
+	arpthreshold := float64(arpcache) * 0.9
+
 	for path, template := range r.templates {
 		file, err := os.Create(path)
 		if err != nil {
 			return fmt.Errorf("error creating config file %s: %v", path, err)
 		}
 
+		items := len(r.serviceUnits)
+		if items > int(arpthreshold) {
+			glog.Warningf("Number of endpoints: %d is exceeding size of ARP neighbour cache threshold: %d", items, int(arpthreshold))
```

Comment:
The wording might be better as:
  The number of distinct endpoints (%d) is close to the size of the ARP neighbor cache (%d).  Please increase the ARP neighbor cache size.

Reply:
And I really only want this message to be logged once.  So we should use the same variable as the one that logs an error reading the arp cache to determine if we've logged already.  If we have, we can entirely skip this check.